business/checkers/authorization/no_host_checker.go

```diff
@@ -16,7 +17,6 @@ type NoHostChecker struct {
 	Namespace           string
 	Namespaces          models.Namespaces
 	ServiceEntries      map[string][]string
```

Comment:
The RegistryServices should include the "ServiceEntries" as well, then perhaps the use of the "ServiceEntries" may be reconsidered ? I mean the host will be also listed as an "External" Service in the RegistryServices.

Reply:
I was thinking to leave this for another PR, as it will extend the scope of this PR.